src/frontend/src/components/public/PIAFormTabs/intake/index.tsx

```diff
@@ -35,6 +35,19 @@ export const PIAFormIntake = () => {
       : stateChangeHandler(false, 'hasAddedPiToDataElements');
   };
 
+  const validatePIOption = () => {
+    if (pia?.hasAddedPiToDataElements === null) {
+      return "I'm not sure";
+    }
+    if (pia?.hasAddedPiToDataElements === true) {
+      return 'Yes';
+    }
+    if (pia?.hasAddedPiToDataElements === false) {
+      return 'No';
+    }
+    return '';
+  };
```

Comment:
can we please leverage "PIOptions" here? No independent string comparisons

Reply:
somewhat resolved